scripts/genpinctrl/config.yaml

```diff
@@ -0,0 +1,139 @@
+# Configuration file for pinctrl generation (except F1 series).
+#
+# This file contains a list of pin configuration templates used to generate the
+# pinctrl files. Each entry can have the following fields:
+#
+#   - name (mandatory): This is the pin alternate function name, e.g. UART_TX.
+#     It is used to group pin configurations alphabetically in the generated
+#     pinctrl files.
+#
+#   - match (mandatory): This is a regular expression used to match against
+#     CubeMX pin configuration names. The regular expression should be as
+#     precise as possible. Note that it needs to be escaped here in the
+#     configuration file.
+#
+#   - bias (optional): Bias setting (disable, pull-up, pull-down). If not set
+#     pinctrl file will not contain the bias entry, equivalent to disable
+#     setting.
```

Comment:
```suggestion
#   - bias (optional): Bias setting (disable, pull-up, pull-down). This setting
#     only applies to "input" mode. Equivalent to "disable" (a.k.a floating) if not. 
```

Reply:
There is no "input" mode for non-F1 series, so I removed `This setting only applies to "input" mode`